PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+Please ensure your pull request adheres to the following guidelines: 
```

Comment:
cf. my comment on README.md, I would add a paragraphe for external contributors, such as:
* please explain your motives to contribute this change: what problem you are trying to fix, what improvement you are trying to make
* link to external resource on how to make good contribution to open source projects